dist/form.js

```diff
@@ -16,6 +16,10 @@ form.addEventListener('submit', e => {
     feedBack.textContent = `${fullName} is  inValid`
     feedBack.style.borderColor = '#fc0b03';
   }
-   
+
+    // set local storage
+    localStorage.setItem('fullName', fullName);
+    localStorage.setItem('email', email);
+    localStorage.setItem('text', text);
```

Comment:
### Nice attempt!!!
- [ ] Your local storage is not perfectly working fine now, although data gets saved, but here are a few things that could be improved:
  - [ ] Form is not submitted when the user clicks on the `Get in touch` button
  - [ ] Data from the local storage is not automatically filling available inputs spaces when the page gets a `onLoad` event, even though it has it in memory.
_Kindly watch this [video](https://watch.screencastify.com/v/eo7cVKP1OmKKF0LBbt4R) to see how the feature is behaving at the moment pls :pleading_face:_

- Here's a tip: Please remember that you need a corresponding eventListener to trigger the retrieval (`localStorage.getItem('obj_name', JSON.parse()`) of data from the local storage.